providers/providertest/provider.go

```diff
@@ -158,7 +158,7 @@ func (o *TestingOracle) RunMarketMap(ctx context.Context, mm mmtypes.MarketMap,
 			if len(prices) != expectedNumPrices {
 				return nil, fmt.Errorf("expected %d prices, got %d", expectedNumPrices, len(prices))
 			}
-			o.Logger.Info("provider prices", zap.Any("prices", prices))
```

Comment:
Is this being removed because it was making the logs too messy?

Reply:
yep